docs/File-Formats.md

```diff
@@ -608,6 +612,13 @@ The MAF format recognized by the portal (excluding the annotation columns alread
 34. **t_ref_count (Optional)**: Reference allele count (tumor).
 35. **n_alt_count (Optional)**: Variant allele count (normal).
 36. **n_ref_count (Optional)**: Reference allele count (normal).
+37. **cbp_driver_passenger (Optional)**: "PUTATIVE_PASSENGER" or "PUTATIVE_DRIVER".
+38. **cbp_driver_passenger_annotation (Optional)**: field to give more information about the cbp_driver_passenger value (limited to 80 characters). This field can only be present if the cbp_driver_passenger is also present in the MAF file. This field is free text. Example values for this field are: "Pathogenic" or "VUS".
+39. **cbp_driver_class (Optional)**: free label/category that marks the mutation as a putative driver (limited to 20 characters). This field is free text. Example values for this field are: "Tier 1", Tier 2"...
+40. **cbp_driver_class_annotation (Optional)**: field to give more information about the cbp_driver_class value (limited to 80 characters). This field can only be present if the cbp_driver_class is also present in the MAF file. This field is free text. Example values for this field are: "Highly Actionable", "Potentially Actionable", "Not currently actionable"...
+
+The "cbp_driver" columns are used in the oncoprint, providing two customized annotations of driver and passenger mutations:
+![schreenshot mutation color menu](images/screenshot-mutation-color-menu.png) 
```

Comment:
I also have a design question regarding the class column. A user might have these classes in his `cbp_driver_class` column:
- Confirmed driver
- Unknown
- Confirmed passenger

If he would have these 3 values in this column, 'Confirmed Passenger' would show up as class for a Driver in the OncoPrint. That wouldn't make sense right? In that case, would he have to replace all 'Confirmed passenger'-values with NA? If so, I think it's important to explain this in `39. **cbp_driver_class (Optional)**:` and/or the text below. 

@pieterlukasse what do you think?